biowdl.wdl

```diff
@@ -73,3 +73,48 @@ task InputConverter {
         json: {description: "JSON file version of the input sample sheet."}
     }
 }
+
+task IndexFastaFile {
+    input {
+        File inputFile
+        String javaXmx = "2G"
+        String memory = "3GiB"
+    }
+    String outputFile = basename(inputFile)
+    # Capture .fa¸ .fna and .fasta
+    String outputDict = sub(outputFile, "\.fn?as?t?a?$", "") + ".dict"
+    # This executes both picard and samtools, so indexes are co-located in the same folder.
+    command <<<
+        set -e
+        cp ~{inputFile} ~{outputFile}
+        picard -Xmx~{javaXmx} \
+            -XX:ParallelGCThreads=1 \
+            CreateSequenceDictionary \
+            REFERENCE=~{inputFile} \
+            OUTPUT="~{outputDict}"
+        samtools faidx ~{outputFile} --fai-idx ~{outputFile}.fai
+    >>>
+
+    output {
+        File outputFasta = outputFile
+        File outputFastaDict = outputDict
+        File outputFastaFai = outputFile + ".fai"
+    }
+
+    runtime {
```

Comment:
time_minutes is missing, our current config defaults to 120, which seems excessive for this task